build/build_env.sh

```diff
@@ -12,7 +12,7 @@ set -e
 ################################################################################
 
 
-export _C89_LSYSLIB="CEE.SCEELKED:SYS1.CSSLIB:CSF.SCSFMOD0"
+export _C89_LSYSLIB="CEE.SCEELKED:SYS1.CSSLIB:PP.CSF.ZOS203.SCSFMOD0"
```

Comment:
Is this compatible with a typical LPAR?

Reply:
No, it's a local change that I needed on my test LPAR.  I don't want that change to be pushed to git.

Reply:
I will revert line 15 to its original in Git.  The build needs `PP.CSF.ZOS203.SCSFMOD0` only on the target z/OS system.